Makefile

```diff
@@ -46,6 +44,12 @@ stop:
 	docker-compose down && \
 	popd
 
+reset:
+	pushd ${radish34} && \
+	docker-compose down && \
+	docker volume rm radish34_mongo-buyer radish34_mongo-supplier1 radish34_mongo-supplier2 radish34_mongo-merkle-tree-volume radish34_chaindata && \
+	popd
```

Comment:
probably worthwhile to also prune any stale networks and images as well?

Reply:
The networks are pruned as part of `docker-compose down`. I added `docker container prune -f` to this command